tasks/main.yml

```diff
@@ -6,6 +6,7 @@
     - libgtk2.0-dev
     - libjasper-dev
     - libjpeg-dev
+    - liblapacke-dev
```

Comment:
As far as I can tell, a lapack headers package. I've linked to an [opencv qa thread](http://answers.opencv.org/question/121651/fata-error-lapacke_h_path-notfound-when-building-opencv-32/) in one of my commit messages, which contains more information around this.

I figured the least intrusive way to solve the issue would be to install the header package, it shouldn't affect any other packages I think.